package.json

```diff
@@ -4,7 +4,8 @@
   "description": "customer reviews for steam toast",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Non-blocking, but I'm guessing this should be `mocha spec` something similar - probably a good idea to add it to save you the hassle of typing it all the time.